Lib/timeit.py

```diff
@@ -150,15 +150,18 @@ def print_exc(self, file=None):
         The optional file argument directs where the traceback is
         sent; it defaults to sys.stderr.
         """
-        import linecache, traceback
+        import _colorize, linecache, traceback, sys
```

Comment:
```suggestion
        import linecache, traceback
```

You can defer the `_colorize` import